lib/src/tabs-nav/Tab.tsx

```diff
@@ -128,6 +145,19 @@ const TabIconContainer = styled.div<TabIconContainerProps>`
   svg {
     height: 24px;
     width: 24px;
+    color: ${(props) => props.theme.unselectedIconColor};
+  }
+
+  &[aria-selected="true"] {
+    svg {
+      color: ${(props) => props.theme.selectedIconColor};
+    }
+  }
+
+  &[aria-disabled="true"] {
+    svg {
+      color: ${(props) => props.theme.disabledIconColor};
+    }
   }
```

Comment:
Disabled and selected styles are not being applied to the icon. Please change these styles to `TabIconContainer` styled component.

```
&[aria-selected="true"] {
    svg {
      color: ${(props) => props.theme.selectedIconColor};
    }
  }

  &[aria-disabled="true"] {
    svg {
      color: ${(props) => props.theme.disabledIconColor};
    }
  }
```

Also, `aria-disabled` is not present to the Tab, so it should be added to the TabContainer as `aria-disabled={disabled}`.